src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -2301,4 +2301,110 @@ namespace winrt::Microsoft::Terminal::Control::implementation
             }
         }
     }
+
+    void ControlCore::AnchorContextMenu(const til::point viewportRelativeCharacterPosition)
+    {
+        // viewportRelativeCharacterPosition is relative to the current
+        // viewport, so adjust for that:
+        _contextMenuBufferPosition = _terminal->GetViewport().Origin() + viewportRelativeCharacterPosition;
+    }
+
+    void ControlCore::_contextMenuSelectMark(
+        const til::point& pos,
+        const std::function<bool(const DispatchTypes::ScrollMark&)>& filter,
+        const std::function<til::point_span(const DispatchTypes::ScrollMark&)>& getSpan)

```

Comment:
This can also be achieved using regular function pointers like so:
```cpp
void ControlCore::_contextMenuSelectMark(
    const til::point& pos,
    bool(*)(const DispatchTypes::ScrollMark&) filter,
    til::point_span(*)(const DispatchTypes::ScrollMark&) getSpan
)
```

The remaining code should still compile the same. This makes it easier on the compiler (`std::function` is a very large, very very complex template, has lots of exception handling routines and is 64 bytes large) and easier on the debugger, which will have a very easy time traceing raw function pointers.

If you'd like to I'd be happy to help you with such a change.

Reply:
I'm assuming these comments went unaddressed